src/lib/CollectStore.js

```diff
@@ -145,7 +145,7 @@ export default class CollectStore {
               arguments: '0 0 0 * * *',
               worker: 'konnector',
               worker_arguments: {
-                konnector: connection.konnector.attributes.slug,
+                konnector: connection.konnector.slug,
```

Comment:
Is it still working the first time ? Maybe we may strengthen our code by using `connection.konnector.attributes ? connection.konnector.attribute.slug : connection.konnector.slug`.

This issue should be resolved soon by standardazing all client js return objects formats.